src/winuwp/runner_uwp/CMakeLists.txt

```diff
@@ -0,0 +1,127 @@
+﻿cmake_minimum_required (VERSION 3.8)
+set(CMAKE_SYSTEM_NAME WindowsStore)
+set(CMAKE_SYSTEM_VERSION 10.0)
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED YES)
+
+include(CMakePrintHelpers)
+
+project (runner LANGUAGES CXX)
+
+# UWP tile and icon assets.
+set(ASSET_FILES ${ASSET_FILES}
+	Assets/LargeTile.scale-100.png
+	Assets/LargeTile.scale-125.png
+	Assets/LargeTile.scale-150.png
+	Assets/LargeTile.scale-200.png
+	Assets/LargeTile.scale-400.png
+	Assets/LockScreenLogo.scale-200.png
+	Assets/SmallTile.scale-100.png
+	Assets/SmallTile.scale-125.png
+	Assets/SmallTile.scale-150.png
+	Assets/SmallTile.scale-200.png
+	Assets/SmallTile.scale-400.png
+	Assets/SplashScreen.scale-100.png
+	Assets/SplashScreen.scale-125.png
+	Assets/SplashScreen.scale-150.png
+	Assets/SplashScreen.scale-200.png
+	Assets/SplashScreen.scale-400.png
+	Assets/Square44x44Logo.altform-unplated_targetsize-16.png
+	Assets/Square44x44Logo.altform-unplated_targetsize-32.png
+	Assets/Square44x44Logo.altform-unplated_targetsize-48.png
+	Assets/Square44x44Logo.altform-unplated_targetsize-256.png
+	Assets/Square44x44Logo.scale-100.png
+	Assets/Square44x44Logo.scale-125.png
+	Assets/Square44x44Logo.scale-150.png
+	Assets/Square44x44Logo.scale-200.png
+	Assets/Square44x44Logo.scale-400.png
+	Assets/Square44x44Logo.targetsize-16.png
+	Assets/Square44x44Logo.targetsize-24.png
+	Assets/Square44x44Logo.targetsize-24_altform-unplated.png
+	Assets/Square44x44Logo.targetsize-32.png
+	Assets/Square44x44Logo.targetsize-48.png
+	Assets/Square44x44Logo.targetsize-256.png
+	Assets/Square150x150Logo.scale-100.png
+	Assets/Square150x150Logo.scale-125.png
+	Assets/Square150x150Logo.scale-150.png
+	Assets/Square150x150Logo.scale-200.png
+	Assets/Square150x150Logo.scale-400.png
+	Assets/StoreLogo.png
+	Assets/StoreLogo.scale-100.png
+	Assets/StoreLogo.scale-125.png
+	Assets/StoreLogo.scale-150.png
+	Assets/StoreLogo.scale-200.png
+	Assets/StoreLogo.scale-400.png
+	Assets/Wide310x150Logo.scale-200.png
+	Assets/WideTile.scale-100.png
+	Assets/WideTile.scale-125.png
+	Assets/WideTile.scale-150.png
+	Assets/WideTile.scale-200.png
+	Assets/WideTile.scale-400.png
+)
+
+# Configure package manifest file.
+set(APP_MANIFEST_NAME Package.appxmanifest)
+set(APP_MANIFEST_TARGET_LOCATION ${CMAKE_CURRENT_BINARY_DIR}/${APP_MANIFEST_NAME})
+set(SHORT_NAME ${BINARY_NAME})
+set(PACKAGE_GUID "F941A77F-8AE1-4E3E-9611-68FBD3C62AE8")
```

Comment:
We'll want to replace this with `{{package_guid}}`

Reply:
When you do that, this file will need to be renamed `CMakeLists.txt.tmpl`

Reply:
I would prefer not to break this from working standalone as part of the review.. maybe can do that in a branch or as part of the porting?